app/views/responsible_body/devices/home/show.html.erb

```diff
@@ -1,10 +1,10 @@
 <%- title = t('page_titles.responsible_body_devices_home') %>
 <%- content_for :title, title %>
 <%- content_for :before_content do %>
-  <% breadcrumbs([
-    { 'Home' => responsible_body_home_path },
-    title,
-  ]) %>
+  <%= render GovukComponent::Breadcrumbs.new(breadcrumbs:{
```

Comment:
do we need to get rid of the syntactic sugar? (`breadcrumbs` is much punchier than `render GovukComponent::Breadcrumbs.new(breadcrumbs:`), and the `"#{title}": ''` could also be cleaner?

Reply:
@benilovj I've taking another stab at replacing the breadcrumbs. I've kept the view helper method as we discussed. I think this is good for another review and potential merge.